pkg/clui/config.go

```diff
@@ -20,11 +20,9 @@ type printConfig struct {
 type printType int
 
 const (
-	printTypeOutput printType = iota
+	printTypeUnknown printType = iota
```

Comment:
**[megacheck]** <sub>reported by [reviewdog](https://github.com/haya14busa/reviewdog) :dog:</sub>
 const printTypeUnknown is unused (U1000)